Dockerfile

```diff
@@ -20,12 +20,14 @@ RUN yum update -y \
 
 # The version in the above image is ancient, and does not support the
 # --no-binary flag used in our requirements files.
-RUN pip install -U pip
+# We also need to install wheels.
+RUN pip install -U pip wheel
 
 COPY requirements /pip/requirements/
 RUN cd /pip && \
     pip install --build ./build --cache-dir ./cache \
-	--no-deps -r requirements/docker.txt && \
+        --find-links https://pyrepo.addons.mozilla.org/ \
+        --no-deps -r requirements/docker.txt && \
```

Comment:
Do you want to add `--find-links https://pyrepo.addons.mozilla.org/wheelhouse/ --no-index` to mimic what we're doing in [the Makefile](https://github.com/mozilla/olympia/blob/master/Makefile#L74)?